src/main/java/io/ebean/config/dbplatform/oracle/OracleDbEncrypt.java

```diff
@@ -13,12 +13,11 @@
  *
  * <pre class="code">
  *
- *  // Remember your DB user needs execute privilege on DBMS_CRYPTO
```

Comment:
Best to NOT autoformat the entire project - leave that to me (as it makes your PR bigger than it needs to be to get 'effective changes' in).